k8s/namespaces/fees-pay/ccpay-cpo-update-service/prod.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: helm.fluxcd.io/v1
+kind: HelmRelease
+metadata:
+  name: ccpay-cpo-update-service
+spec:
+  releaseName: ccpay-cpo-update-service
+  values:
+    global:
+      environment: prod
```

Comment:
this is specified globally for all of the environment, it won't show up if you only build your app though. this shouldn't be included here

Reply:
I've removed the '  values:
    global:
      environment: prod'
Hopefully that is what you meant that should be removed from there